ctl/src/command.rs

```diff
@@ -505,7 +505,7 @@ pub fn status(mut channel: Channel<ConfigMessage,ConfigMessageAnswer>) {
   }
 }
 
-pub fn metrics(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>) {
+pub fn metrics(mut channel: Channel<ConfigMessage,ConfigMessageAnswer>) {
```

Comment:
please do not make unrelated changes when sending a pull request. Even if they can be small cosmetic changes, it makes the PR harder to figure out, and I have a limited time to review them.

Reply:
Sure, I'll split that commit into a separate PR.